conda_smithy/feedstock_tokens.py

```diff
@@ -663,3 +681,25 @@ def add_feedstock_token_to_azure(user, project, feedstock_token, clobber):
             definition_id=ed.id,
             project=ed.project.name,
         )
+
+
+def add_feedstock_token_to_github_actions(
+    user, project, feedstock_token, clobber
+):
+    from .github import gh_token
+    from github import Github
+
+    gh = Github(gh_token())
+    repo = gh.get_repo(f"{user}/{project}")
+
+    if not clobber:
+        status, headers, data = repo._requester.requestJson(
```

Comment:
I am commenting that this looks like the best solution here and that I am surprised pygithub doesn't have an option to list secrets.